lib/iex/resources/resource.rb

```diff
@@ -17,6 +17,41 @@ def self.float_to_percentage(float_number)
       def self.to_dollar(amount:, ignore_cents: true)
         MoneyHelper.money_to_text(amount, 'USD', nil, no_cents: ignore_cents)
       end
+
+      def self.to_full_issue_type(type)
+        case type
+        when 'ad'
+          'ADR'
+        when 're'
+          'REIT'
+        when 'ce'
+          'Closed end fund'
+        when 'si'
+          'Secondary Issue'
+        when 'lp'
+          'Limited Partnerships'
+        when 'cs'
+          'Common Stock'
+        when 'et'
+          'ETF'
+        when 'wt'
+          'Warrant'
+        when 'rt'
+          'Right (blank) – Not Available, i.e., Note, or (non-filing) Closed Ended Funds'
```

Comment:
Just for information. Where did you find these descriptions? I didn't see them in the documentation.

Reply:
https://iexcloud.io/docs/api/#company

>refers to the common issue type of the stock.
ad – American Depository Receipt (ADR’s)
re – Real Estate Investment Trust (REIT’s)
ce – Closed end fund (Stock and Bond Fund)
si – Secondary Issue
lp – Limited Partnerships
cs – Common Stock
et – Exchange Traded Fund (ETF)
wt – Warrant
rt – Right
(blank) – Not Available, i.e., Note, or (non-filing) Closed Ended Funds
ut - Unit
temp - Temporary

We are using the value here.

The type consists of a place called the issue type.
It itself is in other methods and may or may not be described.

The reason why I noticed this is that when I made an error with something that didn't have a KEY, that's where the error occurred.

So, I don't think it's necessary to include the explanation of type.

It's all about the convenience of the IEX, too.

And I don't need this method.

Reply:
I'm sorry, but I had to remove the commit.